README.md

```diff
@@ -67,6 +67,45 @@ Example:
   "rerunOf": "f68f39f9-279c-4e8d-ac38-1216dffcc59c"
 ```
 
+### Report static attributes 
+To report attributes you need to do the following:
+1. Add PublicReportingAPI into your test file. 
+```javascript
+const PublicReportingAPI = require('reportportal-client/lib/publicReportingAPI');
```

Comment:
I advise to re-export this functionality by the agent itself.
Reason: not every API may be implemented, so user will have set of not working functions. Also, I don't like an idea of exposing client itself. Let users work with the agent only.
```
const PublicReportingAPI = require('reportportal-agent-jasmine/...');
// ...
test('...', () => {
  PublicReportingAPI.addAttributes(...)
});
```

Reply:
@AmsterGet what do you think?

Reply:
Definitely yes.
This was one of the points of such a decision.

Reply:
Decided to leave re-export approach